install_config/configuring_sdn.adoc

```diff
@@ -77,21 +77,24 @@ link:../install_config/master_node_configuration.html[node configuration file]
 ----
 networkConfig:
   mtu: 1450 <1>
-  networkPluginName: "redhat/openshift-ovs-subnet" <2>
+  networkPluginName: "" <2>
 ----
 <1> Maximum transmission unit (MTU) for the pod overlay network
-<2> Set to *redhat/openshift-ovs-subnet* for the *ovs-subnet* plug-in or
-*redhat/openshift-ovs-multitenant* for the *ovs-multitenant* plug-in
+<2> Optional for OpenShift SDN plug-in(*redhat/openshift-ovs-subnet* or *redhat/openshift-ovs-multitenant*).
+Set this field if you are using third party network plug-in.
 ====
 
-[[migrating-between-sdn-plugins]]
-== Migrating Between SDN Plug-ins
+[[migrating-to-openshift-sdn-plugins]]
+== Migrating to OpenShift SDN Plug-ins
 
-If you are already using one SDN plug-in and want to switch to another:
+If you are already using some SDN plug-in and want to switch to OpenShift SDN plugin:
 
 . Change the `*networkPluginName*` parameter on all
-link:#configuring-the-pod-network-on-masters[masters] and
+link:#configuring-the-pod-network-on-masters[masters] in their configuration files.
```

Comment:
We switched to xrefs and will only use links for external links, so:
s/link:#configuring-the-pod-network-on-masters[masters]/xref:configuring-the-pod-network-on-masters[masters]